### DISCUSSION TITLE
Can not use python in dolphinscheduler

### DISCUSSION THREAD

**Discussion by LZH-Work-Station** [apache/dolphinscheduler#7734] [Q&A]
The error is : sudo: /opt/soft/python/bin/python: command not found.
I have modified the dolphinscheduler_env.sh file and do install.sh. Java, spark , flink, hive, shell can be used but only python don't work.

**Comment by kezhenxu94** [comment]
Do you have python installed and `/opt/soft/python/bin/python` is executable in your system?

**Reply by LZH-Work-Station** [reply]
no, I think it is the default configuration of dolphinscheduler. I have modified the dolphinscheduler_env.sh for a true path of PYTHON_HOME, but the configuration has not been enabled.

**Reply by kezhenxu94** [reply]
@LZH-Work-Station which version do you use?

**Reply by LZH-Work-Station** [reply]
> @LZH-Work-Station which version do you use?

1.3.8
